packages/sveltekit/test/server/handle.test.ts

```diff
@@ -229,6 +229,7 @@ describe('handleSentry', () => {
       expect(_span!.spanContext().traceId).toEqual('1234567890abcdef1234567890abcdef');
       expect(spanToJSON(_span!).parent_span_id).toEqual('1234567890abcdef');
       expect(spanIsSampled(_span!)).toEqual(true);
+      expect(_span!.metadata!.dynamicSamplingContext).toEqual({});
```

Comment:
hmm, this actually works? 😅 I thought this should fail as this would not be set?

Can we do instead `getDynamicSamplingContextForSpan(_span)` here to test this, maybe?

Reply:
I was as surprised as you that it works 😅 hmm I don't wanna use `getDynamicSamplingContextForSpan` here because I don't think this is necessarily all the logic that contributes to what we ultimately propagate/put into the envelope. Maybe I just refactor these tests similarly to the ones in `load.test.ts` to use the `beforeEnvelope` hook to check for the `trace` header.